### ISSUE TITLE
summary() does not work on CASTable

### ISSUE THREAD

**Issue by qi-yue** [sassoftware/R-swat#14]
Hi folks,

I got an error message when calling the overloaded summary function on a CASTable. Could you take a look when you have time?

Thanks,
Yue

```
> library("swat")
NOTE: The extension module for binary protocol support is not available.
      Only the CAS REST interface can be used.
SWAT 1.2.0.9000
> conn <- CAS('rdcgrdc.unx.sas.com', 63596)
NOTE: Connecting to CAS and generating CAS action functions for loaded
      action sets...
NOTE: To generate the functions with signatures (for tab completion), set 
      options(cas.gen.function.sig=TRUE).
> iris_ct <- as.casTable(conn,iris,casOut = list(replace = TRUE))
NOTE: Cloud Analytic Services made the uploaded file available as table IRIS in caslib CASUSERHDFS(sasyqi).
> head(iris_ct)
  Sepal.Length Sepal.Width Petal.Length Petal.Width Species
1          5.1         3.5          1.4         0.2  setosa
2          4.9         3.0          1.4         0.2  setosa
3          4.7         3.2          1.3         0.2  setosa
4          4.6         3.1          1.5         0.2  setosa
5          5.0         3.6          1.4         0.2  setosa
6          5.4         3.9          1.7         0.4  setosa
> summary(iris_ct)
Error in freqres$results$Frequency %>% select(Column, FmtVar, Frequency) %>%  : 
  could not find function "%>%"
```

**Comment by kesmit13** [comment]
Do you have dplyr installed?

**Comment by qi-yue** [comment]
Yes, I have it installed. I found that I have to explicitly load it 
```
library(dplyr)
```
Otherwise, the summary() function does not work.

**Comment by kesmit13** [comment]
A fix for this was pushed with commit 6513c04f98496db2d69bb46f7a54643c5825e2e5.
